src/operator/numpy/np_elemwise_broadcast_op_extended.cc

```diff
@@ -201,16 +201,12 @@ MXNET_OPERATOR_REGISTER_NP_BINARY_SCALAR(_npi_copysign_scalar)
 
 MXNET_OPERATOR_REGISTER_NP_BINARY_SCALAR(_npi_rcopysign_scalar)
 .set_attr<FCompute>("FCompute<cpu>", BinaryScalarOp::Compute<cpu, mshadow_op::rcopysign>)
-.set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseIn{"_backward_npi_rcopysign_scalar"});
+.set_attr<nnvm::FGradient>("FGradient", MakeZeroGradNodes);
```

Comment:
Why change the grad nodes?

Reply:
Because `rcopysign_grad` is 0 :-): https://github.com/apache/incubator-mxnet/blob/master/src/operator/mshadow_op.h#L777

I noticed this when adding the RTC version, so figured instead of adding a custom code for zero it is better to reuse the existing mechanism to handle the case when gradients are 0.